packages/@aws-cdk/aws-ssm/lib/parameter.ts

```diff
@@ -94,6 +95,18 @@ export interface ParameterOptions {
    * @default - a name will be generated by CloudFormation
    */
   readonly parameterName?: string;
+
+  /**
+   * Determines the separator used to render the ARN for this SSM parameter.
+   * Valid values are `"/"` or `""`.
+   *
+   * If `parameterName` is a path (i.e. begins with "/"), the separator must be
+   * an empty string `""`, otherwise, it must be `"/"`.
+   *
+   * @default - automatically determined based on the value of `parameterName`
+   * unless it is a token, in which case this field is required.
+   */
+  readonly parameterArnSeparator?: string;
```

Comment:
Why is this not simply a boolean `startsWithSlash` or not?

Reply:
I don't know. I am dumb?

Reply:
Will fix.

Reply:
Please fix!! I am getting **Unable to determine ARN separator for SSM parameter since the parameter name is an unresolved token. Use "fromAttributes" and specify "simpleName" explicitly**

Reply:
Can you please raise a new issue with a retro?